MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Selection in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Selection in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Selection in .NET MAUI Segmented (SfSegmented) control
```

Comment:
Selection in .NET MAUI Segmented Control (SfSegmentedControl)

Reply:
changed heading